tests/functional/adapter/custom_schema_tests/test_custom_schema.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from dbt.tests.util import (
+    check_relations_equal,
+    run_dbt
+)
+from tests.functional.adapter.custom_schema_tests.seeds import seed_queries
+
+_VIEW_1_SQL = """
+select * from {{ target.schema }}.seed
```

Comment:
ref('seed')

Reply:
It's failing the test if I do so.

Reply:
```
E           dbt.exceptions.TargetNotFoundError: Compilation Error
E             Model 'model.test.view_1' (models/view_1.sql) depends on a node named 'seed' which was not found

```

Reply:
Interesting. That feels odd. I wonder why we'd need to specify the schema in the model instead of let dbt resolve it.

Reply:
Oh, I bet because the seeds get populated via sql statements instead of as actual seed files. So the models get loaded before the test setup begins. So they don't exist yet, even though they will.

Reply:
I'm curious why the integ tests had it in this way though.

Reply:
Honestly, they were like this because integration tests often had no single standard.